ci/terraform/oidc/storage-token-jwk.tf

```diff
@@ -0,0 +1,60 @@
+module "oidc_storage_token_jwk_role" {
+  source      = "../modules/lambda-role"
+  environment = var.environment
+  role_name   = "oidc-storage-token-jwk-role"
+  vpc_arn     = local.authentication_vpc_arn
+
+  policies_to_attach = [
+    aws_iam_policy.storage_token_kms_signing_policy.arn
+  ]
+}
+
+module "storage_token_jwk" {
+  source = "../modules/endpoint-module"
+
+  endpoint_name   = "storage-token-jwk.json"
```

Comment:
@TaherKapasi are you good with this? Your key will be available on `oidc.(environment.)account.gov.uk/storage-token-jwk.json`